src/views/RecIM/components/List/index.jsx

```diff
@@ -66,15 +69,84 @@ const ParticipantList = ({
 };
 
 const MatchList = ({ matches, activityID }) => {
+  const [selectedDay, setSelectedDay] = useState(0);
+  useEffect(() => {
```

Comment:
More for curiosity here, but do we have to u?e a useEffect? Also, what happens if the index is -1? Wouldn't we want it to go to the next available day with matches instead of just the first tab?

Reply:
Hmmm, this is a good point. The only thing about that is that I struggle with thinking of an implementation that would be able to handle all edge cases without doing a 7 day for-loop to check to make sure there aren't future matches. I could look into it.

This is definitely an implementation I would want and within the scope of this PR. I'll try to tackle it today and ask for re-review!

Reply:
handled